futaba/cogs/roles/core.py

```diff
@@ -265,6 +301,129 @@ async def role_unjoinable(self, ctx, *roles: RoleConv):
             "joinable/remove", ctx.guild, content, icon="role", roles=roles
         )
 
+    @staticmethod
+    def str_channels(channels):
+        return " ".join(f"`{channel.name}`" for channel in channels)
+
+    @role.command(name="pingable")
+    @commands.guild_only()
+    @permissions.check_mod()
+    async def role_pingable(self, ctx, role: RoleConv, *channels: TextChannelConv):
+        logger.info(
+            "Making role '%s' pingable in guild '%s' (%d), channel(s) [%s]",
+            role.name,
+            ctx.guild.name,
+            ctx.guild.id,
+            self.str_channels(channels),
+        )
+
+        if not role:
```

Comment:
Not needed, discord.py will give a missing argument if this isn't there or will fail if it can not match it to a role

Reply:
Done